src/libs/thorvg/tvgSwMemPool.cpp

```diff
@@ -84,7 +84,7 @@ SwMpool* mpoolInit(uint32_t threads)
 {
     auto allocSize = threads + 1;
 
-    auto mpool = static_cast<SwMpool*>(calloc(sizeof(SwMpool), 1));
+    auto mpool = static_cast<SwMpool*>(calloc(1, sizeof(SwMpool)));
```

Comment:
```suggestion
    auto mpool = static_cast<SwMpool*>(calloc(sizeof(SwMpool), 1));
```

Reply:
Removed that commit and submitted https://github.com/thorvg/thorvg/pull/2321.

Reply:
Thanks!